src/MatBlazor/Components/MatAutocompleteList/MatAutocompleteList.razor

```diff
@@ -5,11 +5,23 @@
 
 
 <div class="@WrapperClassMapper.AsString()">
-    <MatTextField Icon="@Icon" OnFocus="@OpenPopup" HideClearButton="true" FullWidth="@FullWidth" OnFocusOut="@ClosePopup" Label="@Label" Value=@StringValue OnInput=@OnValueChanged OnKeyDown="@OnKeyDown" Outlined="@Outlined" Attributes="@Attributes" Id="@Id"></MatTextField>
+    <MatTextField Icon="@Icon"
+                  OnFocus="@OpenPopup"
+                  HideClearButton="true"
+                  FullWidth="@FullWidth"
+                  OnFocusOut="@ClosePopup"
+                  Label="@Label"
+                  Value=@StringValue
+                  ValueExpression="() => StringValue"
+                  OnInput=@OnValueChanged
+                  OnKeyDown="@OnKeyDown"
+                  Outlined="@Outlined"
+                  Attributes="@Attributes"
+                  Id="@Id"></MatTextField>
     @if (IsShowingClearButton)
     {
         <div class="mat-autocomplete-list-clearbutton">
-            <MatIconButton Icon="clear" OnMouseDown="@ClearText"></MatIconButton>
+            <MatIconButton Icon="clear" type="button" OnMouseDown="@ClearText"></MatIconButton>
```

Comment:
@sandrohanea doesn't this need to be updated to the PR #326 

`<MatIconButton Icon="clear" type="button" OnClick="@ClearText"></MatIconButton>`1

Reply:
Not actually...That PR was to enable the value clear with keyboard input but this fix is to not submit the EditForm(In case one exists) when clearing the value: https://github.com/SamProf/MatBlazor/issues/315